packages/carbon-web-components/.env

```diff
@@ -0,0 +1 @@
+PERCY_TOKEN=29af7ff6983fd556c95489ffb282f5a3f286cfe10ad0a4bc492dd4fdcd831539
```

Comment:
should this be pushed up?

Reply:
Oh wow, how did that get in there? Good catch!